.githooks/pre-push

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+cd diagnostics-app
+npm run script:ci
+cd ..
```

Comment:
Can we do something like `npm run diagnostics-app/script:ci`?

Reply:
`npm` commands can only be executed in a folder which contains `package.json` so we have to first `cd` into the directory.